client/src/components/DataFiles/DataFilesBreadcrumbs/DataFilesBreadcrumbs.scss

```diff
@@ -85,12 +68,12 @@
   padding-left: 20px;
 }
 
-#go-to-button-dropdown .complex-dropdown-item-root,
+.complex-dropdown-item-root,
 .complex-dropdown-item-project {
-  display: flex !important;
+  display: flex;
 }
```

Comment:
**Why?**
- Classes are custom and unique, so no need to scope them.[^1]
- The style works without `!important`.

[^1]: Ideally, the classes are modular from `DataFilesBreadcrumbs.module.css` but I'm trying to keep this diff light.